src/config/locales/en-US.json

```diff
@@ -248,7 +249,8 @@
     },
     "feedback_success": "Email successfully open",
     "feedback_fail": "Email client could not open",
-    "server_fail": "Server not available"
+    "server_fail": "Server not available",
+    "show_imgs": "Show images"
```

Comment:
perhaps it should be `Show Images` with (i) capital... most apps use text this way in settings...